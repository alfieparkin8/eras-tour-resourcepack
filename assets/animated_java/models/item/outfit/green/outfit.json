{
	"parent": "animated_java:item/outfit/outfit",
	"textures": {
		"2": "animated_java:item/outfit/green_layer_1",
		"4": "animated_java:item/outfit/green_layer_1",
		"6": "animated_java:item/outfit/green_layer_2"
	}
}