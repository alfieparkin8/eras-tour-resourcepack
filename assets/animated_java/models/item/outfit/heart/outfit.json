{"parent":"animated_java:item/outfit/outfit","textures":{"2":"minecraft:block/outfits/lover/heart_layer_1"}}