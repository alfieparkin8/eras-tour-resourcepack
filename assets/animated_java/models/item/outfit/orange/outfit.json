{"parent":"animated_java:item/outfit/outfit","textures":{"2":"minecraft:block/outfits/1989/orange_layer_1","4":"minecraft:block/outfits/1989/orange_layer_1","6":"minecraft:block/outfits/1989/orange_layer_2"}}