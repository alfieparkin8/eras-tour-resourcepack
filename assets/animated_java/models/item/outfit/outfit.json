{
	"textures": {
		"2": "minecraft:block/outfits/lover/sunset_layer_1"
	},
	"elements": [
		{
			"from": [
				3.25,
				-2.75,
				5.25
			],
			"to": [
				12.75,
				10.75,
				10.75
			],
			"faces": {
				"north": {
					"uv": [
						5,
						10,
						7,
						16
					],
					"texture": "#2"
				},
				"east": {
					"uv": [
						7,
						10,
						8,
						16
					],
					"texture": "#2"
				},
				"south": {
					"uv": [
						8,
						10,
						10,
						16
					],
					"texture": "#2"
				},
				"west": {
					"uv": [
						4,
						10,
						5,
						16
					],
					"texture": "#2"
				},
				"up": {
					"uv": [
						7,
						10,
						5,
						8
					],
					"texture": "#2"
				},
				"down": {
					"uv": [
						9,
						10,
						7,
						8
					],
					"texture": "#2"
				}
			}
		}
	],
	"display": {
		"head": {
			"rotation": [
				0,
				180,
				0
			]
		}
	}
}