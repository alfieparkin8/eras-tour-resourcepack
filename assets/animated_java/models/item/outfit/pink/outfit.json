{
	"parent": "animated_java:item/outfit/outfit",
	"textures": {
		"2": "animated_java:item/outfit/pink_layer_1",
		"4": "animated_java:item/outfit/pink_layer_1",
		"6": "animated_java:item/outfit/pink_layer_2"
	}
}