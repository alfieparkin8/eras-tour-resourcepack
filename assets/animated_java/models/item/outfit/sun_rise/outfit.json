{
	"parent": "animated_java:item/outfit/outfit",
	"textures": {
		"2": "minecraft:block/outfits/lover/tangerine_layer_1",
		"4": "minecraft:block/outfits/lover/tangerine_boots_layer_1"
	}
}