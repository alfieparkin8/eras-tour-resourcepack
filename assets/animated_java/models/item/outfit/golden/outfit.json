{"parent":"animated_java:item/outfit/outfit","textures":{"0":"minecraft:block/outfits/lover/golden_layer_1","1":"minecraft:block/outfits/lover/heart_layer_1","2":"minecraft:block/outfits/lover/golden_layer_1","3":"minecraft:block/outfits/lover/violet_layer_1","4":"minecraft:block/outfits/lover/golden_boots_layer_1","5":"minecraft:block/outfits/lover/golden_boots_layer_1","6":"minecraft:block/outfits/blank","7":"minecraft:block/outfits/1989/blue_layer_1","8":"minecraft:block/outfits/1989/blue_layer_2","9":"minecraft:block/outfits/1989/green_layer_1","10":"minecraft:block/outfits/1989/green_layer_2","11":"minecraft:block/outfits/1989/orange_layer_1","12":"minecraft:block/outfits/1989/orange_layer_2","13":"minecraft:block/outfits/1989/pink_layer_1","14":"minecraft:block/outfits/1989/pink_layer_2"}}